_data/compatibility/wasabi.yaml

```diff
@@ -10,17 +10,17 @@ rbf:
     version: "1.0"
   features:
     receive:
-      notification: "na"
+      notification: "false"
       list: "false"
       details: "false"
       shows_replaced_version: "true"
       shows_original_version: "false"
     send:
       signals_bip125: "false"
-      list: "untested"
-      details: "untested"
-      shows_replaced_version: "untested"
-      shows_original_version: "untested"
+      list: "false"
```

Comment:
If sending an RBF signaling transaction isnt possible from Wasabi, there is no way to tell what a "rbf-signaling-sent-from-wasabi" transaction would look like in the list/details pages. That is why, in these scenarios we have the fields as untested. Let me know if that is unclear.

Reply:
One can test this by importing the seed to Electrum and sending RBFs and see that the tx is properly replaced.